app/views/users/edit.html.slim

```diff
@@ -54,6 +54,11 @@
 
   = f.collection_check_boxes :interested_in, User::INTERESTS.to_a, :first, :last, item_wrapper_tag: 'div', item_wrapper_class: 'interested_in'
 
+  h3.page-header Technologies:
+
+  = f.input :tech_expertise_list, placeholder: 'Languages, techniques, etc', label: 'Technologies of expertise', hint: 'Separate tags by comma.'
```

Comment:
How about also phrasing things from a "me"-perspective here? Same as with the other form fields in the user profile?

Maybe something along
> Programming Languages, Frameworks, Technologies, etc. I'm experienced with

and 
> Other Languages, Technologies, etc. I'm interested in

???

Plus, even though we treat these things like "tags" under the hood, for the user it's "just" a text input 😉 
So I would suggest to change the hint to a simple "separate with comma".